heudiconv/tests/test_bids.py

```diff
@@ -1116,3 +1126,32 @@ def test_BIDSFile():
     # -for an existing entity, you can overwrite it with "set":
     my_bids_file.set('echo', '2')
     assert my_bids_file['echo'] == '2'
+
+
+@pytest.mark.skipif(not have_datalad, reason="no datalad")
+def test_ME_mag_phase_conversion(tmpdir, subID='MEGRE', heuristic='bids_ME'):
+    """ Unit test for the case of multi-echo GRE data with
+    magnitude and phase.
+    The different echoes should be labeled automatically.
+    """
+    tmpdir.chdir()
+    tmppath = tmpdir.strpath
+    try:
+        datadir = fetch_data(tmppath, "dicoms/velasco/{subID}")
```

Comment:
may be
```suggestion
        datadir = fetch_data(tmppath, f"dicoms/velasco/{subID}")
```
would explain why test seems to be skipping?  
why would you want to skip anyways if you had datalad installed (there is `@skipif`) , so let's just do without skipping and corrected (will give another bigger suggestion)